app/src/main/java/org/ometa/lovemonster/ui/activities/UserLoveActivity.java

```diff
@@ -75,7 +75,12 @@ private void setupMakeLoveButton() {
         makeLoveButton.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
-                final MakeLoveDialogFragment makeLoveDialogFragment = MakeLoveDialogFragment.newInstance(user);
+                final MakeLoveDialogFragment makeLoveDialogFragment;
+                if (user.equals(currentUser)) {
+                    makeLoveDialogFragment = MakeLoveDialogFragment.newInstance(new User("foo@example.com", ""));
```

Comment:
Agreed.  But, it's 11 o'clock :)

On Mon, Dec 7, 2015 at 10:58 PM Anthony Caliendo <notifications@github.com>
wrote:

> In
> app/src/main/java/org/ometa/lovemonster/ui/activities/UserLoveActivity.java
> <https://github.com/lovemonster/love-monster-android/pull/33#discussion_r46915179>
> :
>
> > @@ -75,7 +75,12 @@ private void setupMakeLoveButton() {
> >          makeLoveButton.setOnClickListener(new View.OnClickListener() {
> >              @Override
> >              public void onClick(View v) {
> > -                final MakeLoveDialogFragment makeLoveDialogFragment = MakeLoveDialogFragment.newInstance(user);
> > +                final MakeLoveDialogFragment makeLoveDialogFragment;
> > +                if (user.equals(currentUser)) {
> > +                    makeLoveDialogFragment = MakeLoveDialogFragment.newInstance(new User("foo@example.com", ""));
>
> That constraint is probably overly restrictive, considering we only ever
> really need a username for stuff to work.
>
> Probably is safe to remove the requirement on email address being there,
> and only require username.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lovemonster/love-monster-android/pull/33/files#r46915179>
> .
>